oauthenticator/oauth2.py

```diff
@@ -72,6 +72,10 @@ def _OAUTH_ACCESS_TOKEN_URL(self):
     def _OAUTH_USERINFO_URL(self):
         return self.authenticator.userdata_url
 
+    @property
+    def _OAUTH_EXTRA_AUTHORIZE_PARAMS(self):
```

Comment:
Is this needed? The comment above says "these URLs are part of the OAuth2Mixin API get them from the Authenticator object", but this property isn't listed in https://www.tornadoweb.org/en/stable/auth.html#tornado.auth.OAuthMixin

Reply:
No, it isn't needed. I originally added it to be consistent with the other properties. Looking at this again, I think you're right and it's probably better to remove it.